### PR TITLE
RPi4 WiFi support

### PR DESCRIPTION
**Pull request by mydatascience**
#149 
This functionality adds the Wifi support to RPI4. 
Additional parameter of SSID is added. Then WiFi is supported instead of eth0 interface. Eth0 is switched off.
Use Wifi config via:
`./eden config add default --devmodel RPi4 --ssid WIFISSID  `

This will be updated in the future to allow all interfaces at once.

### CODE REVIEW

File: cmd/certs.go [hunk 1]
```diff
@@ -67,6 +71,19 @@ var certsCmd = &cobra.Command{
 		if err := utils.GenerateSSHKeyPair(filepath.Join(certsDir, "id_rsa"), filepath.Join(certsDir, "id_rsa.pub")); err != nil {
 			log.Fatal(err)
 		}
+		if ssid != "" && password != "" {
```
Comment (rvs) [inline cmd/certs.go:74]: What about cases with open ssid? (not that those are common nowadays, but still)

File: pkg/controller/devModel.go [hunk 2]
```diff
@@ -44,10 +44,13 @@ func (ctx *DevModel) GetNetNoDHCPID() string {
 
 //SetWiFiParams set ssid and psk for RPI
 func (ctx *DevModel) SetWiFiParams(ssid string, psk string) {
+	if ctx.devModelType != defaults.DefaultRPIModel {
```
Comment (rvs) [inline pkg/controller/devModel.go:47]: Why is this only enabled for RPI? Ultimately this leads to generation of DevicePortConfig and it seems orthogonal to RPi -- it is just is a useful piece of functionality to have.

File: cmd/certs.go (hunk 1 above)
Comment (giggsoff) [reply cmd/certs.go:74]: In this PR we add support only for WiFiKeyScheme_WPAPSK. We can add WiFiKeyScheme_WPAEAP and WiFiKeyScheme_SchemeNOOP in next PRs.

File: pkg/controller/devModel.go (hunk 2 above)
Comment (giggsoff) [reply pkg/controller/devModel.go]: Done. I have added more common variant into this function.

**Comment by rvs** [comment]
LGTM
